mne/cov.py

```diff
@@ -386,66 +389,90 @@ def compute_raw_covariance(raw, tmin=None, tmax=None, tstep=0.2,
     picks : array-like of int | None (default None)
         Indices of channels to include (if None, all channels
         except bad channels are used).
+    keep_sample_mean : bool (default False)
```

Comment:
> what do you mean it was False before?

This must be the source of confusion. The way `compute_raw_covariance` works in `master` is equivalent to using `compute_raw_covariance(..., keep_sample_mean=False)` in the current PR. So proposing to have no `keep_sample_mean` arg and have it behave like `keep_sample_mean=True` means you can no longer get a computation equivalent to current `master` for `compute_raw_covariance`. Make sense?